lib/facade_converter.ts

```diff
@@ -283,6 +283,31 @@ export class FacadeConverter extends base.TranspilerBase {
         this.emit(')');
       });
     },
+    'Array.forEach': (c: ts.CallExpression, context: ts.Expression) => {
```

Comment:
It should.
The only advantage of having the check here it that it would fail earlier.